source/develop/release-management/features/network/detailed-osn-integration.html.md

```diff
@@ -183,7 +183,7 @@ The integration of network providers into oVirt will be incremental. The followi
 
 ### User Experience
 
-![ thumb | right](Providers.png  "fig: thumb | right") ![ thumb | right](Discovery.png  "fig: thumb | right") ![ thumb | right](Add.png  "fig: thumb | right") ![ thumb | right](Networks.png  "fig: thumb | right")
+![ thumb | right](/images/wiki/Providers.png) ![ thumb | right](/images/wiki/Discovery.png) ![ thumb | right](/images/wiki/Add.png) ![ thumb | right](/images/wiki/Networks.png)
```

Comment:
More ` thumb | right` issues.

To script it, I guess it would be possible to take the filename, de-camel-case it and add " screenshot" afterward. That would probably be a decent start?